File: sample/DataContext.cs
using Glow.Sample.Files;
using Microsoft.EntityFrameworkCore;

namespace Glow.Sample
{
    public class DataContext : DbContext
    {
        public DataContext(DbContextOptions<DataContext> options) : base(options) { }

        public DbSet<PortfolioFile> PortfolioFiles { get; set; }
        public DbSet<Portfolio> Portfolios { get; set; }
        public DbSet<MdxBundle.Entity> MdxBundle { get; set; }

        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            base.OnModelCreating(modelBuilder);
            modelBuilder.Entity<Portfolio>()
                .Property(v => v.RowVersion)
                .IsRowVersion();
        }
    }
}

File: sample.fs/DataContext.fs
namespace sample.fs

open  Microsoft.EntityFrameworkCore;

type Person =
    { PersonId : int
      FirstName : string
      LastName : string
      Address : string
      City : string}

type DataContext(options: DbContextOptions<DataContext>) =
    inherit DbContext(options)

    [<DefaultValue>]
    val mutable persons : DbSet<Person>

    member public this.Persons with get() = this.persons
                               and set p = this.persons <- p

